python/ray/WebUI.ipynb

```diff
@@ -1,10 +1,46 @@
 {
  "cells": [
   {
+   "cell_type": "markdown",
+   "metadata": {
+    "hideCode": true,
+    "hidePrompt": false
+   },
+   "source": [
+    "### Ray UI."
```

Comment:
Instead of making the title and the instructions two separate markdown boxes, let's make them a single markdown box.